src/utils/MessageBuilder.ts

```diff
@@ -0,0 +1,222 @@
+        })
+        .join("\n");
+
+      embed
+        .setTitle(`Current Queue: ${ballchasers.length}/6`)
+        .setDescription("Click the green button to join the queue! \n\n" + ballChaserList);
+    }
+
+    return {
+      components:
+        getEnvVariable("ENVIRONMENT") == "dev"
+          ? [new MessageActionRow({ components: [joinButton, leaveButton, fillTeamButton, removeAllButton] })]
+          : [new MessageActionRow({ components: [joinButton, leaveButton] })],
+      embeds: [embed],
+    };
+  }
+
+  static fullQueueMessage(ballchasers: ReadonlyArray<Readonly<PlayerInQueue>>): MessageOptions {
+    const embed = new MessageEmbed({
+      color: "GREEN",
+      thumbnail: { url: this.normIconURL },
+    });
+    const pickTeamsButton = new MessageButton({
+      customId: ButtonCustomID.CreateRandomTeam,
+      label: "Create Teams",
+      style: "SUCCESS",
+    });
+    const leaveButton = new MessageButton({
+      customId: ButtonCustomID.LeaveQueue,
+      label: "Leave",
+      style: "DANGER",
+    });
+    const removeAllButton = new MessageButton({
+      customId: ButtonCustomID.RemoveAll,
+      label: "DEV: Remove All",
+      style: "DANGER",
+    });
+
+    const ballChaserList = ballchasers
+      .map((ballChaser) => {
+        // + 1 since it seems that joining the queue calculates to 59 instead of 60
+        const queueTime = ballChaser.queueTime?.diffNow().as("minutes") ?? 0;
+        return `${ballChaser.name} (${Math.min(queueTime + 1, 60).toFixed()} mins)`;
+      })
+      .join("\n");
+
+    embed
+      .setTitle("Queue is Full")
+      .setDescription("Click the Create Teams button to get started! \n\n" + ballChaserList);
+
+    return {
+      components:
+        getEnvVariable("ENVIRONMENT") == "dev"
+          ? [new MessageActionRow({ components: [pickTeamsButton, leaveButton, removeAllButton] })]
+          : [new MessageActionRow({ components: [pickTeamsButton, leaveButton] })],
+      embeds: [embed],
+    };
+  }
+
+  static activeMatchMessage(ballchasers: ReadonlyArray<Readonly<PlayerInQueue>>): MessageOptions {
+    const embed = new MessageEmbed({
+      color: "#A62019",
+      thumbnail: { url: this.normIconURL },
+    });
+    const reportMatch = new MessageButton({
+      customId: ButtonCustomID.ReportMatch,
+      label: "Report Match",
+      style: "SUCCESS",
+    });
+    const breakMatch = new MessageButton({
+      customId: ButtonCustomID.BreakMatch,
+      label: "DEV: Break Match",
+      style: "DANGER",
+    });
+
+    const orangeTeam: Array<string> = [];
+    const blueTeam: Array<string> = [];
+
+    ballchasers.forEach((ballChaser) => {
+      if (ballChaser.team === Team.Blue) {
+        blueTeam.push("<@" + ballChaser.id + ">");
+      } else {
+        orangeTeam.push("<@" + ballChaser.id + ">");
+      }
+    });
+
+    embed
+      .setTitle("Teams are set!")
+      .addField("🔶 Orange Team 🔶", orangeTeam.join("\n"))
```

Comment:
Blue team should be before orange team to match what default Rocket League lobby settings.

Reply:
Fixed in next commit.